aspnet/mvc/views/partial/sample/src/PartialViewsSample/Views/Shared/Error.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿@{
```

Comment:
Not super sure what the purpose of this page is. Wouldn't this be part of the Diagnostics sample?